src/main/java/com/homeworks/factorymethodd/Creator.java

```diff
@@ -0,0 +1,5 @@
+package com.homeworks.factorymethodd;
+
+public abstract class Creator {
+    public abstract void factoryMethod();
```

Comment:
Should this method be void or it should return an object?

Reply:
I think it could be void